### DISCUSSION TITLE
Stereo Tool info...

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#5923] [Q&A] [answered]
Looking at your newish implementation of Stereo Tool, I find the basic info somewhat underwhelming;

• You warn about ensuring sufficient CPU & memory resources without specifying a minimum 
• Binary 9.91 appears to be installed, but no dashboard is present to configure it
• Nowhere is it stated that once activated one must upload a config. file and no explanation on how such a file is to be generated.

Thanks for clearing this up a bit

**Comment by VParvulus** [comment]
Have you ever used Stereo Tool or even just googled it?

You have to use the software made by Thimo to create a config for it https://www.thimeo.com/stereo-tool/ there is no "dashboard" apart from that.

Stereo Tool can do so many things and chain a lot of stuff together so I wouldn't expect any mentions of specific cpu and memory use here. I mean, how should someone know this stuff beforehand without knowing how you configure this tool? 😆

**Reply by SpeedyCharly** [reply]
Sorry to say, but you don't seem to know much about the inner-workings of puters...

**Reply by VParvulus** [reply]
Says the guy calling computers "puters" 😆 

Your comment history looks awesome btw. always antagonizing people who interact with your threads like a Karen.
Must be nice being the center of the world mate.

**Comment by BusterNeece** [comment, marked as answer]
You install your own Stereo Tool binary. It's a "power user" tool intended for folks who are familiar with it already and have a configured preset they want to use; the purpose is to make it easier to integrate that custom configuration into an AzuraCast station. Otherwise, we don't provide direct support or any closer integration of Stereo Tool.

**Reply by SpeedyCharly** [reply]
Thanks Buster;  As the other respondent so rudely noted, I have no direct experience with Stereo Tool, but it would still be nice if we had an idea how much server space, CPU/memory resources are needed to make it work alongside Azuracast... just so we know beforehand if that's an avenue worth exploring for us...

**Reply by VParvulus** [reply]
You sound like a whiney kid who is told to be a sensible human and at least do some research for yourself instead of being fed everything on a silver plate.

An audio processor that is used to apply dozens of effects and transformations on a stream is not something you only explore/consider when you know beforehand what kind of resource usage it has (which you can't define explicitly since it highly depends on what transformers and processors you actually apply). You look at it because you want your audio signal to be processed to a certain degree to have it sound a certain way and then scale your resources accordingly.

If resources are such a humongous issue that you absolutely need to know them beforehand then test it on your staging environment instead of testing directly on production. You know, like someone who knows what they do actually does when exploring tech they have no idea about yet.

**Comment by dougllasvieira** [comment]
StereoTool is a great tool but it is a CPU-eater, a 1 core vps and 1GB of ram will not serve you well.

StereoTool has its own web interface in the stand alone version, but the DSP/VST version I can't remember now. 

These are the recommended settings if you want to use it:
4vCore
4GB Ram

**Reply by SpeedyCharly** [reply]
Thanks - That's the info I was looking for!
